src/amqp10/messageHeader.ts

```diff
@@ -12,7 +12,7 @@ export class Header {
           const formatCode = dataResponse.readUInt8()
           dataResponse.rewind(1)
           const decodedBoolean = decodeFormatCode(dataResponse, formatCode)
-          if (!decodedBoolean) throw new Error(`invalid formatCode %#02x: ${formatCode}`)
+          if (decodedBoolean === undefined) throw new Error(`invalid formatCode %#02x: ${formatCode}`)
```

Comment:
Why? I strongly suggest to never check directly `undefined` or `null` I think the previous code is much better

Reply:
The problem is that in the previous code even if the decodedBoolean is false it throws an exception, but false is correct